go.mod

```diff
@@ -485,6 +485,8 @@ replace github.com/microcosm-cc/bluemonday => github.com/microcosm-cc/bluemonday
 // Needed due to a deprecated method used in functional tests
 replace github.com/stretchr/testify => github.com/stretchr/testify v1.8.4
 
+replace github.com/diagridio/go-etcd-cron => github.com/joshvanl/go-etcd-cron v0.0.0-20240901031816-5173b9308d3e
```

Comment:
Can this be updated to the latest release?

Reply:
Yep! Done, thanks :+1: